src/develop/blend_gui.c

```diff
@@ -1217,6 +1340,124 @@ gboolean blend_color_picker_apply(dt_iop_module_t *module, GtkWidget *picker, dt
   else return FALSE; // needs to be handled by module
 }
 
+static gboolean _blendif_change_blend_colorspace(dt_iop_module_t *module, dt_develop_blend_colorspace_t cst)
+{
+  switch(cst)
+  {
+    case DEVELOP_BLEND_CS_RAW:
+    case DEVELOP_BLEND_CS_LAB:
+    case DEVELOP_BLEND_CS_RGB_DISPLAY:
+    case DEVELOP_BLEND_CS_RGB_SCENE:
+      break;
+    default:
+      cst = dt_develop_blend_default_module_blend_colorspace(module);
+      break;
+  }
+  if(cst != module->blend_params->blend_cst)
+  {
+    dt_develop_blend_init_blendif_parameters(module->blend_params, cst);
+
+    dt_iop_gui_blend_data_t *bd = module->blend_data;
+    const int cst_old = _blendop_blendif_get_picker_colorspace(bd);
+    dt_dev_add_history_item(darktable.develop, module, FALSE);
+    dt_iop_gui_update(module);
+
+    if(cst_old != _blendop_blendif_get_picker_colorspace(bd) &&
+       (gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(bd->colorpicker)) ||
+        gtk_toggle_button_get_active(GTK_TOGGLE_BUTTON(bd->colorpicker_set_values))))
+    {
+      dt_iop_color_picker_set_cst(bd->module, _blendop_blendif_get_picker_colorspace(bd));
+      dt_dev_reprocess_all(bd->module->dev);
+      dt_control_queue_redraw();
+    }
+
+    return TRUE;
+  }
+  return FALSE;
+}
+
+static void _blendif_select_colorspace(GtkMenuItem *menuitem, dt_iop_module_t *module)
+{
+  dt_develop_blend_colorspace_t cst = GPOINTER_TO_INT(g_object_get_data(G_OBJECT(menuitem), "dt-blend-cst"));
+  if(_blendif_change_blend_colorspace(module, cst))
+  {
+    gtk_widget_queue_draw(module->widget);
+  }
+}
+
+static void _blendif_options_callback(GtkButton *button, GdkEventButton *event, dt_iop_module_t *module)
+{
+  if(event->button != 1 && event->button != 2) return;
+
+  GtkWidget *mi;
+  GtkMenu *menu = darktable.gui->presets_popup_menu;
+  if(menu) gtk_widget_destroy(GTK_WIDGET(menu));
+  darktable.gui->presets_popup_menu = GTK_MENU(gtk_menu_new());
+  menu = darktable.gui->presets_popup_menu;
+
+  // add a section to switch blending color spaces
+  const dt_develop_blend_colorspace_t module_cst = dt_develop_blend_default_module_blend_colorspace(module);
+  const dt_develop_blend_colorspace_t module_blend_cst = module->blend_params->blend_cst;
+  if(module_cst == DEVELOP_BLEND_CS_LAB || module_cst == DEVELOP_BLEND_CS_RGB_DISPLAY
+      || module_cst == DEVELOP_BLEND_CS_RGB_SCENE)
+  {
+    char *markup;
+
+    mi = gtk_menu_item_new_with_label(_("default blend colorspace"));
```

Comment:
IIUC, this reset the blending to the default for the module, right?

So maybe use : "reset to default blend colorspace" or "select default blend colorspace" ?